components/SeaCreatureCard.jsx

```diff
@@ -49,12 +41,7 @@ export default function SeaCreatureCard({
     return (
       <StyledOverlay>
         <SeaCreatureDetailedCard
-          name={name}
-          imageUri={imageUri}
-          catchPhrase={catchPhrase}
-          speed={speed}
-          shadow={shadow}
-          price={price}
+          seaCreature={seaCreature}
           hue={hue}
           onClick={setShowDetailedCard}
           id={id}
```

Comment:
Hier das gliche wie oben, du nimmst die ID in der Detailcard nicht mal mehr entgegen, also bitte raus